dev/com.ibm.ws.jpa_22_fat/test-applications/cdi/src/cdi/web/ELIServlet.java

```diff
@@ -312,6 +313,74 @@ public void testInjectionOccursBeforeInsertAndRemoveCallbacks() throws Exception
         svLogger.logp(Level.INFO, CLASS_NAME, TNAME, TNAME + "().exit");
     }
 
+    @Test
+    @Mode(TestMode.LITE)
```

Comment:
It isn't necessary to annotate for `@Mode(LITE)`, since it is the default test mode.  The only time you'd ever need to annotate for LITE is if `@Mode(FULL)` was declared at the class level (thus making all tests FULL mode), and want to override a certain set of individual tests to be LITE mode.

Reply:
Updated